foundry/entity.d.ts

```diff
@@ -169,13 +176,13 @@ declare class Entity<D extends Entity.Data = Entity.Data> {
    * actor.sheet // ActorSheet
    * ```
    */
-  get sheet(): BaseEntitySheet<BaseEntitySheet.Data<this>, this> | null;
+  get sheet(): FormApplication<FormApplication.Data<this>, this> | null;
```

Comment:
Are there entities which don't use the `BaseEntitySheet`?

Reply:
Not sure anymore why I changed it. I will change it back.